kubernetes/linera-validator/build_and_redeploy.sh

```diff
@@ -96,15 +105,15 @@ else
     helm install linera-core . --values values-local.yaml --wait --set installCRDs=true || exit 1;
 fi
 
-echo "Pods:";
-kubectl get pods;
-echo -e "\nServices:";
-kubectl get svc;
+echo "Pods:"
+kubectl get pods
+echo -e "\nServices:"
+kubectl get svc
 
-docker rm linera-test-local;
-docker run -d --name linera-test-local "$docker_image" \
-    && docker cp linera-test-local:wallet.json /tmp/ \
-    && docker cp linera-test-local:linera.db /tmp/
```

Comment:
TBH I think that sequences like these in scripts might be better served by `errexit` anyway.